docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6304,6 +6324,380 @@ field [@ProtoAny]. At the moment, there is not any mechanism to extend the
 `p4.v1.TableEntry` message based on the value of architecture-specific table
 properties, but we may include on in future versions of the API.
 
+# GenericTable { #sec-generic-table}
+
+## GenericTable p4info { #sec-p4info-generic-table}
```

Comment:
Suggest using capitalization `P4Info` to be consistent with other occurrences in the spec.  I won't mention it for every occurrence in comments -- please do a global search and replace check on your PR.